src/map.cpp

```diff
@@ -1193,13 +1193,11 @@ bool Map::isBlockOccluded(v3s16 pos_relative, v3s16 cam_pos_nodes, bool simple_c
 	// this is a HACK, we should think of a more precise algorithm
 	u32 needed_count = 2;
 
-	v3s16 random_point(myrand_range(-bs2, bs2), myrand_range(-bs2, bs2), myrand_range(-bs2, bs2));
-	if (!isOccluded(cam_pos_nodes, pos_blockcenter + random_point, step, stepfac,
-				start_offset, end_offset, needed_count))
-		return false;
-
-	if (simple_check)
-		return true;
+	if (simple_check) {
```

Comment:
a quick comment explaining that this is only used on the server and why it doesn't work well on the client would be good.